### PR TITLE
README: updates to include dispatch CLI

### PR DESCRIPTION
**Pull request by achille-roussel**
Working on updating the README to based instructions on using the Dispatch CLI. I have a few updates in here that I think aren't supported yet but I'd like to get to, so I'll investigate updating the SDK first before we merge this.

WIP

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,166 +33,75 @@ Python package to develop applications with the Dispatch platform.
...
-```sh
-pip install dispatch-py
-```
-
-## Usage
+### Installing the Dispatch CLI
 
-The SDK allows Python applications to declare functions that Dispatch can
-orchestrate:
+As a pre-requisite, we recommend to install the Dispatch CLI to simplify the
```
Comment (chriso) [inline README.md:42]:
```suggestion
As a pre-requisite, we recommend installing the Dispatch CLI to simplify the
```

File: README.md [hunk 2]
```diff
@@ -32,166 +33,75 @@ Python package to develop applications with the Dispatch platform.
...
+configuration and execution of applications that use Dispatch. On OSX, this
```
Comment (chriso) [inline README.md:43]:
```suggestion
configuration and execution of applications that use Dispatch. On macOS, this
```

File: README.md [hunk 3]
```diff
@@ -32,166 +33,75 @@ Python package to develop applications with the Dispatch platform.
...
+can be done easily using the [Homebrew](https://docs.brew.sh/) tap:
```
Comment (chriso) [inline README.md:44]:
```suggestion
can be done easily using [Homebrew](https://docs.brew.sh/):
```

File: README.md [hunk 4]
```diff
@@ -32,166 +33,75 @@ Python package to develop applications with the Dispatch platform.
...
-A common approach consists of using [ngrok][ngrok] to setup a public endpoint
-that forwards to the server running on localhost.
-
-For example, assuming the server is running on port 8000 (which is the default
-with FastAPI), the command to create a ngrok tunnel is:
-```sh
-ngrok http http://localhost:8000
-```
-Running this command opens a terminal interface that looks like this:
+dispatch run -- main.py
```
Comment (chriso) [inline README.md:101]:
```suggestion
dispatch run -- python3 main.py
```

We don't have an automatic Python entry point yet.

File: README.md [hunk 5]
```diff
@@ -243,11 +153,63 @@ async def transform(msg):
 ```
 
 Dispatch converts Python coroutines to *Distributed Coroutines*, which can be
-suspended and resumed on any instance of a service across a fleet.
+suspended and resumed on any instance of a service across a fleet. For a deep
+dive on these concepts, read our blog post on
+[*Distributed Coroutines with a Native Python Extension and Dispatch*](https://stealthrocket.tech/blog/distributed-coroutines-in-python).
```
Comment (chriso) [inline README.md:158]:
```suggestion
[*Distributed Coroutines in Python*](https://stealthrocket.tech/blog/distributed-coroutines-in-python).
```

File: README.md [hunk 6]
```diff
@@ -243,11 +153,63 @@ async def transform(msg):
...
+configured to verify that requests originated from Dispatch. These
+configuration options can be passed as arguments to the
+the `Dispatch` constructor, but by default they will be loaded from environment
+variables:
+
+| Environment Variable        | Value Example                      |
+| :-------------------------- | :--------------------------------- |
+| `DISPATCH_API_KEY`          | `d4caSl21a5wdx5AxMjdaMeWehaIyXVnN` |
+| `DISPATCH_ENDPOINT_URL`     | `https://service.domain.com`       |
+| `DISPATCH_VERIFICATION_KEY` | `-----BEGIN PUBLIC KEY-----...`    |
```
Comment (chriso) [inline README.md:208]: Maybe a note here that you don't need to populate these when using the CLI since it does it automatically?

**Review by chriso — APPROVED** [review]

**Comment by achille-roussel** [comment]
The changes in this PR were included in https://github.com/stealthrocket/dispatch-py/pull/154, closing.

**Pull request closed without merging**